ansible/roles/jenkinsslave/tasks/main.yml

```diff
@@ -34,6 +34,7 @@
     owner={{ jenkinsuser }}
     group={{ jenkinsuser }}
     mode=0700
+    remote_src=True
```

Comment:
pointing ``snoopy_dir_path:`` to a local dir did not make any difference if it is what you mean